pkgs/by-name/xm/xmltoman/package.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, perl538Packages
+, perl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xmltoman";
+  version = "0.6";
+
+  src = fetchFromGitHub {
+    owner = "atsb";
+    repo = "xmltoman";
+    rev = version;
+    hash = "sha256-EmFdGIeBEcTY0Pqp7BJded9WB/DaXWcMNWh6aTsZlLg=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    perl
+  ];
+
+  buildInputs = [
+    perl538Packages.XMLTokeParser
+  ];
+
+  buildPhase = ''
```

Comment:
Please split `postPatch` `buildPhase` and `preInstall`.

Reply:
Didn't resolve this. Also overriding phases need adding `runHook`: `runHook preBuild` and `runHook postBuild`

Reply:
Didn't look into this carefully. You can put all these to `installPhase`.